auth/src/main/java/com/firebase/ui/auth/data/remote/SignInKickstarter.java

```diff
@@ -92,9 +92,8 @@ public void onComplete(@NonNull Task<CredentialRequestResponse> task) {
 
     private void startAuthMethodChoice() {
         List<AuthUI.IdpConfig> idpConfigs = getArguments().providerInfo;
```

Comment:
Not sure what you mean. Ctrl + Shift + N lets me search for files. 

I changed it so that there is no local idpConfigs list, we just do getArguments().providerInfo.get(0) instead. Should we add a sanity check here since we access the first element without checking if it exists?

Reply:
Lol, my B. I meant `Ctrl + Alt + N`. 😊 You should try it, I'm always impressed.

Reply:
But yeah, LGTM